src/store/education/slice.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import { IEducation } from 'src/common/types';
+import { addEducation } from './actions';
+
+interface IEducationState {
+  educationExperience: IEducation[];
+}
+
+const initialState: IEducationState = {
+  educationExperience: [],
```

Comment:
I think we can cut it down to just experience

Reply:
...or `education`

Reply:
![image](https://user-images.githubusercontent.com/41474876/160571112-ad314bc5-09a6-443f-b15c-c66825949c21.png)